docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -59,19 +59,30 @@ image::images/view-alert-details.png[Shows the Event Renderer button, 200]
 
-NOTE: If an alert is linked to more than one threat, a compiled version of threat indicator data displays in the *threat summary* section. A more detailed view displays within the *Threat Intel* tab.
+- `file.hash.md5`: The MD5 hash
+- `file.hash.sha1`: The SHA1 hash
+- `file.hash.sha256`: The SHA256 hash
+- `file.pe.imphash`: Imports in a PE file
+- `file.elf.telfhash`: Imports in an ELF file
+- `file.hash.ssdeep`: The SSDEEP hash
+- `source.ip`: The IP address of the source (IPv4 or IPv6)
+- `destination.ip`*: The event's destination IP address
+- `url.full`: The full URL of the event source
+- `registry.path`: The full registry path, including the hive, key, and value
 
-* *Threat Intel*: Shows the number of matched threats and displays them individually. Threats appear in reverse chronological order, with the most recent alerts at the top. The available `threat.indicator` and `source.event` data is displayed for each threat. If the alert has not been enriched with threat data, the *Threat Intel* tab displays the message "No Threat Intel Enrichment Found" and provides a link to Threat Intel module documentation.
+NOTE: To further verify the type of threat intelligence data you're viewing, hover your mouse over the field name. Also note that if an alert is linked to more than one threat, matched field values  display within the *Summary* tab and a more detailed view can be found within the *Threat Intel* tab.
+
+* *Threat Intel*: Shows the number of threats linked the alert and displays indicator rule enrichments and investigation time enrichments individually. Threats appear in reverse chronological order, with the most recent events at the top. Threat titles are formed using matched fields and atomic indicators, and available `threat.indicator` and `source.event` data is displayed for each threat. If the alert does not contain indicator rule enrichments or investigation time enrichments, the *Threat Intel* tab displays the message "No Indicator Match or Threat Intel Enrichment Found" and provides a link to Threat Intel module documentation. Similarly, if the alert does not contain indicator rule enrichments, the *Threat Intel* tab displays the message "No indicator match rule enrichments found" and provides a link to Threat Intel module documentation.
```

Comment:
Missing word. Also, would it help to include the link to Threat Intel docs here, too (mentioned in last sentence)?
```suggestion
* *Threat Intel*: Shows the number of threats linked to the alert and displays indicator rule enrichments and investigation time enrichments individually. Threats appear in reverse chronological order, with the most recent events at the top. Threat titles are formed using matched fields and atomic indicators, and available `threat.indicator` and `source.event` data is displayed for each threat. If the alert does not contain indicator rule enrichments or investigation time enrichments, the *Threat Intel* tab displays the message "No Indicator Match or Threat Intel Enrichment Found" and provides a link to Threat Intel module documentation. Similarly, if the alert does not contain indicator rule enrichments, the *Threat Intel* tab displays the message "No indicator match rule enrichments found" and provides a link to Threat Intel module documentation.
```

Reply:
The copy "No Indicator Match or Threat Intel Enrichment Found" is outdated and now could be one of three states (ref: https://github.com/elastic/kibana/pull/105701); it might be sufficient to mention "a message is displayed" instead of referencing specific copy.